### DISCUSSION TITLE
App name

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5764] [Q&A] [answered]
I started to use statamic but i want to change the APP name on .env file but when i did this it got ERROR, please how can i change the name to be suitable to my project

**Comment by duncanmcclean** [comment, marked as answer]
You changed it in the right place. What error did you get?

If it's multiple words, you may need to surround it in double quotes: eg `APP_NAME="Duncan's Testing Website"`

**Reply by HazemMohamed252** [reply]
Yes the issue was due to i am using two words. Thanks so much
